ui/components/app/detected-token/detected-token.js

```diff
@@ -63,27 +63,30 @@ const DetectedToken = ({ setShowDetectedTokens }) => {
   );
   const currentChainId = useSelector(getCurrentChainId);
   const allNetworks = useSelector(getNetworkConfigurationsByChainId);
-  const tokenNetworkFilter = useSelector(getTokenNetworkFilter);
-  const allOpts = {};
-  Object.keys(allNetworks || {}).forEach((chainId) => {
-    allOpts[chainId] = true;
-  });
 
-  const allNetworksFilterShown =
-    Object.keys(tokenNetworkFilter).length !==
-    Object.keys(allOpts || {}).length;
+  const isTokenNetworkFilterEqualCurrentNetwork = useSelector(
+    getIsTokenNetworkFilterEqualCurrentNetwork,
+  );
 
   const totalDetectedTokens = useMemo(() => {
-    return process.env.PORTFOLIO_VIEW && !allNetworksFilterShown
+    return process.env.PORTFOLIO_VIEW &&
+      !isTokenNetworkFilterEqualCurrentNetwork
```

Comment:
I feel like this diff is the opposite condition from what it was previously, but that this change is the correct one. I think that previously we may have incorrectly named this variable.

Reply:
agree, it was named backwards before